### PR TITLE
Issue #4013: Split and Organize Checkstyle inputsby Test for LineLength

### PR DESCRIPTION
**Pull request by MikhailovOlegM**
#4013

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputSimpleLineLength
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java]: input should be named after it's test.
Test is `LineLengthCheckTest`, input should be named `InputLineLength`....
Please rename class and file.
Please do the same for all other inputs in this test.

Comment (MikhailovOlegM) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java]: Please explain in more detail
I have a resource file - InputSimple.java. It must be renamed to InputSimpleLineLength?
Or it is rename LineLengthCheckTest to rename on InputLineLengthCheckTest?

Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java]: > I have a resource file - InputSimple.java

Must be renamed to `InputLineLengthSimple`.
My main point is it needs to start with `InputLineLength`.

**Comment by rnveach** [comment]
@MikhailovOlegM Read over my example closer. https://github.com/checkstyle/checkstyle/pull/4071#discussion_r107271057

> Must be renamed to `InputLineLengthSimple`.

You made it `InputSimpleLineLength`.

**Comment by romani** [comment]
@MikhailovOlegM , commits in PR should be squashed

**Comment by MikhailovOlegM** [comment]
I have to use `git rebase`?

**Comment by romani** [comment]
`git reabse -i HEAD~3`
please read about this command in web

**Comment by rnveach** [comment]
@MikhailovOlegM ping
Since we are out of proposal deadline, do you think you can finish this?

**Comment by MikhailovOlegM** [comment]
ssory, no
Because my university did not give me a certificate in English and I did
not have time to apply for participation

2017-04-03 22:11 GMT+03:00 rnveach <notifications@github.com>:

**Comment by rnveach** [comment]
@MikhailovOlegM Just to confirm, you will no longer try and finish this PR?

**Comment by MikhailovOlegM** [comment]
yes

**Comment by romani** [comment]
author changes were squashed and merged.

@MikhailovOlegM , do you want to have another issue, even you do not want to participate in current GSoC year, you can keep practicing on project to be ready for next year.

**Pull request closed without merging**